### DISCUSSION TITLE
第2节课笔记(5班)

### DISCUSSION THREAD

**Discussion by sanbuphy** [InternLM/Tutorial#125] [笔记]
宝子们在这里提交你们的笔记

**Comment by ATang0729** [comment]
笔记：https://github.com/ATang0729/InternLMCamp_Notes/blob/main/notes/class2-LLMDemo/note_class2.md

**Comment by Milkve** [comment]
大模型第二节课笔记：[[学习笔记] 轻松分钟玩转书生·浦语大模型趣味 Demo](https://juejin.cn/spost/7320456973899644928)

**Comment by shengmouget** [comment]
https://blog.csdn.net/shengweiit/article/details/135423659

**Comment by JayceNing** [comment]
https://zhuanlan.zhihu.com/p/676406720

**Comment by DreamBinary** [comment]
[https://github.com/DreamBinary/InternLMCamp/tree/master/day02](https://github.com/DreamBinary/InternLMCamp/tree/master/day02)

**Comment by ZhaoHongSen0726** [comment]
csdn：https://blog.csdn.net/m0_75085274/article/details/135425532?spm=1001.2014.3001.5502

**Comment by angellyao** [comment]
我的知乎笔记：https://zhuanlan.zhihu.com/p/676472085

**Comment by Beforcer** [comment]
https://zhuanlan.zhihu.com/p/676490909

**Comment by GitEasonXu** [comment]
第2节：轻松玩转书生·浦语大模型趣味 Demo笔记
https://github.com/GitEasonXu/InternLM/blob/main/homework/course_2/coursenote.md

**Comment by Farewell-CK** [comment]
第二节课笔记：https://github.com/8baby8/Farewell_Learn_puyu/blob/main/%E7%AC%AC%E4%BA%8C%E8%8A%82%E8%AF%BE/%E7%AC%94%E8%AE%B0.md

**Comment by Lengxiaom0** [comment]
第二课节笔记:
https://mp.csdn.net/mp_blog/creation/editor/135455578

**Comment by yilisong007** [comment]
好奇问下，有人实现过在自己PC电脑上安装和运行一种一个demo不？比如说InternLM-Chat-7B 智能对话 Demo。因为我看在开发机上面运行demo的时候，在跟demo服务对话的时候会消耗GPU和GPU的内存，是否说我们一般的笔记本电脑还需要看是否有GPU以及是否跟模型适配之类的问题？

**Comment by preordinary** [comment]
https://blog.csdn.net/preor/article/details/135433974?csdn_share_tail=%7B%22type%22%3A%22blog%22%2C%22rType%22%3A%22article%22%2C%22rId%22%3A%22135433974%22%2C%22source%22%3A%22preor%22%7D

**Comment by yilisong007** [comment]
https://github.com/yilisong007/InternLM_learn/blob/main/class2 笔记

**Comment by Lengxiaom0** [comment]
https://blog.csdn.net/lxmlxmmmm/article/details/135513342?csdn_share_tail=%7B%22type%22%3A%22blog%22%2C%22rType%22%3A%22article%22%2C%22rId%22%3A%22135513342%22%2C%22source%22%3A%22lxmlxmmmm%22%7D
